Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs

```diff
@@ -117,10 +122,38 @@ private Vector2 GetScreenResoultion()
         private void Awake()
         {
             _logger = PatcherLogManager.DefaultLogger;
+            _canvasScaler = GetComponent<CanvasScaler>();
+
+            float scaler;
+            float currentDpi = Screen.dpi;
```

Comment:
I'd recommend renaming it to `float screenDpi`.

Reply:
done